### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by bpepple**
This pull request will add some documentation to the project.

First thing you'll want to do is install the optional doc dependencies. These are needed since poetry is a realitively new project and doesn't have proper support with ReadTheDocs. To add them you need to run `poetry install -E docs`.

To create the documentation locally, you would change to the docs directory and run `make html`. The results can be then found in docs/build/html directory.

If a new module is added to the project, you will need to go to the docs directory and run `sphinx-apidoc -o source ../Simyan` to have that module's documentation added.

To host the documentation at ReadTheDocs, the first thing you need to do is create an account there. Once there you should be able to import the repo. I've already added a readthedoc.yml to the project so there should not be much configuration needed, but if you into any problems just holler.

### CODE REVIEW

**Comment by bpepple** [comment]
Oh, one thing I forgot. The project didn't seem to be using the standard __version__ when the it's being initialized, and for the documentation it's useful to have.  So, when you publish new releases you need to make sure to update the values in the project.toml **&** the project __init__.py.

File: tests/test_issues.py [hunk 1]
```diff
@@ -67,15 +76,18 @@ def test_issue_list(comicvine):
...
 
 
 def test_issue_no_has_staff_review(comicvine):
+    """Test issue that has staff review data."""
     result = comicvine.issue(505513)
     assert "has_staff_review" not in result.__dict__.keys()
 
 
 def test_issue_list_no_has_staff_review(comicvine):
+    """Test issue that does not have staff review data."""
```
[the remark this answers is not in the document]
Comment (bpepple) [reply tests/test_issues.py:91]: Yup, tho the function name should probably be change to `test_issue_list_has_staff_review`

[the remark this answers is not in the document]
Comment (Buried-In-Code) [reply tests/test_issues.py]: correct, and the one about should probably be renamed to `test_issue_has_staff_review` as well to match

**Comment by Buried-In-Code** [comment]
@bpepple are you ok with fixing these conflicts?

**Comment by bpepple** [comment]
I can give them a look later tonight

**Pull request merged**